pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -348,13 +348,21 @@ def _check_config_and_set_final_flags(
                 else:
                     self._cluster_environment_flag = getattr(self._strategy_flag, "cluster_environment")
 
-            # TODO: RFC existing accel_conn doesn't handle this, should we add conflict check?
-            #   eg: parallel_device is torch.device(cpu) but accelerator=gpu
             if hasattr(self._strategy_flag, "parallel_devices"):
```

Comment:
should this logic sit inside of the parallel strategy class as part of the setters for the accelerator/parallel_devices properties?

Reply:
Not really.. we need use the parallel_device type to choose Accelerator in accl_conn, and it will influence strategy fallback logic.